ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-access-lists.j2

```diff
@@ -0,0 +1,20 @@
+{% if ip_access_lists is arista.avd.defined %}
+
+## IP Access-lists
+
+### IP Access-lists Summary
+{%     if ip_access_lists_max_entries is arista.avd.defined %}
+
+> The maximum number of ACL entries allowed to be provisioned per switch: {{ ip_access_lists_max_entries }}
+{%     else %}
+
+> The maximum number of ACL entries allowed to be provisioned per switch is not limited.
+> Set `ip_access_lists_max_entries` to configure a guard if required.
```

Comment:
This should not be in the device documentation. I suggest to remove lines 9-12 and move the `if` in line 6 up to line 4 (above the blank line)